.github/workflows/build_windows.yml

```diff
@@ -79,23 +70,19 @@ jobs:
     - name: Display version information
       run: ./ccextractorwinfull.exe --version
       working-directory: ./windows/Release-Full
-    - name: Prepare artifacts
-      run: mkdir ./windows/artifacts
-    - name: Copy CCExtractor executable for artifact
-      run: cp ./windows/Release-Full/ccextractorwinfull.exe ./windows/artifacts/
-    - name: Copy GUI for artifact
-      run: cp ./windows/Release/ccextractorgui.exe ./windows/artifacts/; 
-    - name: Copy DLL's needed to artifact
-      run: cp ./windows/Release-Full/*.dll ./windows/artifacts/
-    - uses: actions/upload-artifact@v1
+    - uses: actions/upload-artifact@v2
       with:
         name: CCExtractor Windows OCR and HardSubX Release build
-        path: ./windows/artifacts
+        path: |
+          ./windows/Release-Full/ccextractorwinfull.exe
+          ./windows/Release/ccextractorgui.exe
```

Comment:
This triggers the Artifact to have 2 folders. If possible this should be avoided and all files placed in a single folder.

Reply:
@canihavesomecoffee I've tried using the wild card pattern using OR ( { } ), which looks something like this : 
`./windows/{Release/ccextractorgui.exe,Release-Full/{ccextractorwinfull.exe,*.dll}}`
but it doesn't seem to work, since the OR is not yet defined in the github wildcard patterns.
So I tried uploading the files belonging to the same directory first and then uploading the files belonging to the other directory and it worked. This was mentioned in the [updated github actions](https://github.com/actions/upload-artifact#uploading-to-the-same-artifact). The copying of artifacts task is now eliminated.